ert_shared/ensemble_evaluator/entity/snapshot.py

```diff
@@ -126,14 +131,14 @@ def from_cloudevent(cls, event):
         snapshot = cls()
         e_type = event["type"]
         e_source = event["source"]
-        status = _FM_TYPE_EVENT_TO_STATUS[e_type]
         timestamp = event["time"]
+
         if e_type in ids.EVGROUP_FM_STAGE:
             queue_state = event.data.get("queue_event_type")
             snapshot.update_stage(
                 get_real_id(e_source),
                 get_stage_id(e_source),
-                status=status,
+                status=_FM_TYPE_EVENT_TO_STATUS[e_type],
```

Comment:
can you explain the logic here?

Reply:
Removed from the top of the function because _FM_TYPE_EVENT_TO_STATUS[e_type] would fail when e_type is of the new types